src/Tools/dotnet-monitor/Program.cs

```diff
@@ -58,12 +58,12 @@ class Program
                 Argument = new Argument<bool>(name: "metrics", getDefaultValue: () => true )
             };
 
-        private static Option ReversedServerAddress() =>
+        private static Option DiagnosticPort() =>
             new Option(
-                alias: "--reversed-server-address",
-                description: "A fully qualified path and filename for the OS transport to communicate over.")
+                alias: "--diagnostic-port",
+                description: "A fully qualified path and filename of the listening diagnostic port to which runtime instances can connect.")
```

Comment:
```suggestion
                description: "The fully qualified path and filename of the diagnostic port to which runtime instances can connect.")
```

The Diagnostic Port that the tool creates would be considered a Connect Diagnostic Port if you follow the terminology in the runtime.  I'm suggesting we don't surface the Connect vs Listen distinction to users and simply call it the "Diagnostic Port".